soccer/Robot.hpp

```diff
@@ -414,6 +453,7 @@ class OurRobot: public Robot {
 		radioTx.set_sing(true);
 	}
 
+	bool isPenaltyKicker = false;
```

Comment:
Yeah I remember the justification, I just don't like that gameplay stuff is leaking into lower levels :/.  I think it's fine to leave, but if there's a way to avoid it in the future, let's do that.